resources/etc/init.d/openhab2

```diff
@@ -55,44 +85,23 @@ elif [ -r /etc/init.d/functions ]; then
   . /etc/init.d/functions
 fi
 
-# Read configuration variable file if it is present
-if [ -r /etc/default/openhab2 ]; then
-  . /etc/default/openhab2
-fi
-
-OPENHAB_CONF_DIR="/etc/openhab2"
-OPENHAB_DIR="/usr/share/openhab2"
-OPENHAB_HOME_DIR="/var/lib/openhab2"
-OPENHAB_USER_DATA_DIR="${OPENHAB_HOME_DIR}"
-
 if [ -z "$USER_AND_GROUP" ]; then
-  USER_AND_GROUP="openhab:openhab"
+  USER_AND_GROUP="${OPENHAB_USER}:${OPENHAB_GROUP}"
 fi
 
 if [ x"${OPENHAB_JAVA}" != x ]; then
   JAVA="${OPENHAB_JAVA}"
 fi
 
-if [ -n "${EXTRA_JAVA_OPTS}" ]; then
-  export EXTRA_JAVA_OPTS
-fi
-
-if [ -n ${OPENHAB_HTTP_PORT} ]; then
-  export OPENHAB_HTTP_PORT
-fi
-
-if [ -n ${OPENHAB_HTTPS_PORT} ]; then
-  export OPENHAB_HTTPS_PORT
-fi
 
 
 if [ ! -x "${JAVA}" ]; then
   log_failure_msg "no java executable found at ${JAVA}"
   exit 2
 fi
 
-USER=`echo ${USER_AND_GROUP} | cut -d ":" -f 1`
-GROUP=`echo ${USER_AND_GROUP} | cut -d ":" -f 2`
+USER=$(echo ${USER_AND_GROUP} | cut -d ":" -f 1)
+GROUP=$(echo ${USER_AND_GROUP} | cut -d ":" -f 2)
```

Comment:
I've kept it for legacy reasons, someone may have used the $USER_AND_GROUP variable to set both at the same time. If they updated, then they'll run into some unexpected problems.